lib/src/executables/sort_intl.dart

```diff
@@ -0,0 +1,329 @@
+          .split(':')
+          .last
+          .trim()
+  };
+
+  final processedPackages = Set<String>();
+  await pubGetForAllPackageRoots(dartPaths);
+
+  // Is this necessary, or a duplicate of the earlier call? Like, do we have to run
+  // a null codemod again after the pub get?
+  exitCode = await runInteractiveCodemod(
+    [],
+        (_) async* {},
+    args: codemodArgs,
+    additionalHelpOutput: parser.usage,
+  );
+  if (exitCode != 0) return;
+  print('^ Ignore the "codemod found no files" warning above for now.');
+
+  for (String package in packageRoots) {
+    final packageRoot = p.basename(package);
+    final packageName = packageNameLookup[packageRoot] ?? 'fix_me_bad_name';
+    _log.info('Starting migration for $packageName');
+    List<String> packageDartPaths;
+    try {
+      packageDartPaths =
+          dartFilesToMigrateForPackage(package, processedPackages).toList();
+    } on FileSystemException {
+      _log.info('${package} does not have a lib directory, moving on...');
+      return;
+    }
+
+    packageDartPaths = limitPaths(packageDartPaths, allowed: dartPaths);
+    sortPartsLast(packageDartPaths);
+
+    final IntlMessages messages = IntlMessages(packageName,
+        directory: fs.currentDirectory, packagePath: package);
+
+    exitCode =
+    await runMigrators(packageDartPaths, codemodArgs, messages, packageName);
+
+    processedPackages.add(package);
+
+    messages.write(force: parsedArgs[_noMigrate]);
+    messages.format();
+  }
+}
+
+void printUsage() {
+  stderr.writeln(
+      'Migrates literal strings that seem user-visible in the package by wrapping them in Intl.message calls.');
+  stderr.writeln();
+  stderr.writeln('Usage:');
+  stderr.writeln('intl_sorting [arguments]');
+  stderr.writeln();
+  stderr.writeln('Options:');
+  stderr.writeln(parser.usage);
+}
+
+/// Runs a set of codemod sequences separately to work around an issue where
+/// updates from an earlier suggestor aren't reflected in the resolved AST
+/// for later suggestors.
+///
+/// This means we have to set up analysis contexts multiple times, which takes longer,
+/// but isn't a dealbreaker. E.g., for wdesk_sdk, running two sequences takes 2:52
+/// as opposed to 2:00 for one sequence.
+///
+/// If any sequence fails, returns that exit code and short-circuits the other
+/// sequences.
+Future<int> runCodemodSequences(
+    Iterable<String> paths,
+    Iterable<Iterable<Suggestor>> sequences,
+    List<String> codemodArgs,
+    ) async {
+  for (final sequence in sequences) {
+    final exitCode = await runInteractiveCodemodSequence(
+      paths,
+      sequence,
+      defaultYes: true,
+      args: codemodArgs,
+      additionalHelpOutput: parser.usage,
+    );
+    if (exitCode != 0) return exitCode;
+  }
+
+  return 0;
+}
+
+/// Define your migrators here
+
+// Example:
+// final intlPropMigrator = IntlMigrator(messages.className, messages);
+
+Future<int> runMigrators(List<String> packageDartPaths,
```

Comment:
Delete this method.

Reply:
This method is for running migration with only `--no-migrate ` args

Reply:
Yes, and I'm saying we shouldn't run migrators at all.